darwin/importer/importer.py

```diff
@@ -364,13 +373,17 @@ def import_annotations(
         if files_to_track:
             _warn_unsupported_annotations(files_to_track)
             for parsed_file in track(files_to_track):
-                image_id = remote_files[parsed_file.full_path]
+                remote_file = remote_files[parsed_file.full_path]
```

Comment:
`image_id, default_slot_name = remote_files[parsed_file.full_path]`

is cleaner